package.json

```diff
@@ -75,7 +76,7 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -e %HUSKY_GIT_PARAMS%",
+      "commit-msg": "commitlint -e $HUSKY_GIT_PARAMS",
```

Comment:
@gagyibenedek will this work for you or will you have again the Windows environment issues?

Reply:
@lirantal so this was a quick fix with some quick googling as it wont let me push. happy to change it or revert it if another PR fixes it.

Reply:
on the command line you can just do 'git commit --no-verify' and it will pass the HUSKY commands

I pushed a PR now that is supposed to fix this support for both Win and Linux/Mac

Reply:
I'll check it tomorrow!

Reply:
I kind of already merged that change as part of the update-notifier PR :-)